java/src/jmri/jmrit/sensorgroup/Bundle.java

```diff
@@ -0,0 +1,97 @@
+package jmri.jmrit.sensorgroup;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.CheckReturnValue;
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ *
+ * Convention is to provide a subclass of this name in each package, working off
+ * the local resource bundle name.
+ *
+ * @author Bob Jacobsen Copyright (C) 2012
+ * @since 3.3.1
+ */
+public class Bundle extends jmri.jmrit.Bundle {
+
+    @Nullable
+    private static final String name = "jmri.jmrit.sensorgroup.SensorgroupBundle"; // NOI18N
```

Comment:
Recommend `name` be `jmri.jmrit.sensorgroup.Bundle` and the new .properties files be named `Bundle[_locale].properties`. By convention, translatable properties files should be named `Bundle[_locale].properties` (see [JMRI Code: Internationalization](http://jmri.org/help/en/html/doc/Technical/I8N.shtml) under "Adding a new Bundle"). Translatable properties files not so named generally predate the Bundle.java mechanism for internationalization.